dcp-0010/dcp-0010.mediawiki

```diff
@@ -0,0 +1,448 @@
+S<sub>pow</sub> = The subsidy for proof of work at a given height for a given number of votes<br />
+S<sub>vote</sub> = The subsidy for a single vote at a given height<br />
+h = The height for which the subsidy is to be calculated<br />
+v = The number of votes included in the block
+H<sub>sv</sub> = The height at which stake validation begins (4096 on <code>mainnet</code>)<br />
+v<sub>b</sub> = The number of votes per block (5 on all networks)
+v<sub>m</sub> = The minimum number of required votes (3 on all networks)
+
+===Subsidy Enforcement===
+
+IMPORTANT: The PoS subsidy, and hence each individual  vote subsidy, is based on
+the block PRIOR to the one that contains the vote since that prior block is the
+one being voted on.  In practice, this means the formula to calculate the vote
+subsidy MUST be called with a height of one less than the PoW subsidy formula
+once the height that voting begins at is reached.
+
+The following modified rules MUST be enforced:
+
+* The input value of the one and only input of the coinbase transaction MUST commit to the modified PoW subsidy proportion
+* The total of the output values of the coinbase transaction MUST NOT exceed the modified PoW subsidy proportion plus the total transaction fees
+* The input value of the stakebase of vote transactions MUST commit to the modified single vote subsidy proportion
+* The proportional payouts of vote transactions MUST be calculated based on the modified single vote subsidy proportion
+* The total of the output values of vote transactions MUST NOT exceed modified single vote subsidy proportion
+
+==Rationale==
+
+Proof of Work is intended to provide several key properties to Decred, some of
+which are:
+
+* Unambiguous determination of the exact contents of a block
+* Unforgeable historical timestamps
+* A source of entropy
+* Significantly increased cost to conduct a wide variety of attacks
+* Protection against data manipulation and double spends
+* Contributes to unforgeable costliness and irreversibility
+* Fair coin distribution
+
+While PoW has worked well for almost all of these properties, unfortunately
+extensive analysis has shown that the expected results of a fair coin
+distribution has not been achieved.
+
+A commonly overlooked aspect of PoW mining is that it has to behave similarly to
+gold mining in order for it to achieve the expected results of a fair coin
+distribution.  That is the reason the process is referred to as mining.
+
+More concretely, the price is expected to track the cost of production such
+that, as the profitability goes up, the incentives for competition also go up
+which leads to more participants (hash power) and thus profitability reduces
+accordingly.
+
+That expected result has not happened in Decred because there is an effective
+monopoly on the ASICs which has led to Decred significantly overpaying for what
+PoW provides.  The result of that has been a high degree of centralization of
```

Comment:
While it may be the case that "That expected result has not happened in Decred because there is an effective monopoly on the ASICs which has led to Decred significantly overpaying for what PoW provides.", it would perhaps be more pertinent and incontrovertible to just say "That expected result has not happened in Decred, which has led to Decred significantly overpaying for what PoW provides, as evidenced by the relative stability in overall network hashrate throughout long periods of large (e.g. 10-20x) changes in exchange rate."

Reply:
While I think it is effectively saying the same thing, since the only realistic way the observed result happens is when there is a monopoly, I agree that it's better explicitly call out the evidence as opposed to assuming the reader is already familiar enough to make that leap.

Updated.